packages/shared/src/guards.ts

```diff
@@ -420,22 +412,24 @@ export function isComplexValue(
   return type.class === DTypeClass.Complex;
 }
 
-function assertPrimitiveValue(
+function assertScalarValue(
```

Comment:
Forgot to rename this assertion function when I renamed `Primitive` to `ScalarValue`.